packages/core/src/llm-core/utils/similarity.ts

```
@@ -0,0 +1,227 @@
+    ])
+
+    private static readonly REGEX = {
+        chinese: /[\u4e00-\u9fff]/,
+        japanese: /[\u3040-\u30ff\u3400-\u4dbf]/,
+        english: /[a-zA-Z]/
+    }
+
+    private static detectLanguages(text: string): Set<string> {
+        const languages = new Set<string>()
+
+        if (this.REGEX.chinese.test(text)) languages.add('zh')
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)